sklearn/metrics/tests/test_classification.py

```diff
@@ -1599,6 +1602,64 @@ def test_log_loss():
     loss = log_loss(y_true, y_score2, labels=[1, 2, 3])
     assert_almost_equal(loss, 1.0630345, decimal=6)
 
+    # test multi-label #1
+    mlb = MultiLabelBinarizer()
+
+    y_true = [['foo'], ['foo', 'bar'], ['foo'], ['bar']]
+    y_true = mlb.fit_transform(y_true)
+    y_proba = np.asarray([[.6, .5], [.3, .9], [.4, .4], [.8, .6]])
+
+    loss_all_labels = list()
+    for y_true_l, y_proba_l in zip(y_true.T, y_proba.T):
+        loss_binary_label = log_loss(y_true_l, y_proba_l)
+        loss_all_labels.append(loss_binary_label)
+
+    loss = assert_warns_message(ChangedBehaviorWarning, 'multilabel log loss',
+                                log_loss, y_true, y_proba)
+    assert_almost_equal(loss, np.mean(loss_all_labels))
+
+    # test multi-label #2
+    y_true = [['cat', 'dog', 'unicorn'], ['cat'], ['dog', 'unicorn'],
+              ['cat', 'unicorn']]
+    y_true = mlb.fit_transform(y_true)
+    y_proba = np.asarray([[.9, .1, .1],
```

Comment:
Is there something fundamentally different between this and the previous?

Reply:
removed duplicated test case.